kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -1055,9 +1055,11 @@ struct VerifyDbCheckpoint {
     os << "  }";
     return os.str();
   }
-  bool isSame(const Digest &d, const STDigest &st) const {
+
+  bool isSame(const Digest &d, const Digest &st) const {
```

Comment:
Is this function really required?
Can existing operator== be used?

Reply:
`isSame()` removed.